packages/lib-classifier/src/store/TutorialStore.js

```diff
@@ -137,13 +155,26 @@ const TutorialStore = types
       self.activeMedium = undefined
     }
 
+    function resetSeen (type) {
+      if (type === 'tutorial') {
+        self.tutorialSeenTime = undefined
+      } else if (type === 'mini-course') {
+        self.miniCourseSeenTime = undefined
+      } else {
+        self.tutorialSeenTime = undefined
+        self.miniCourseSeenTime = undefined
+      }
+    }
+
     return {
       afterAttach,
       fetchMedia: flow(fetchMedia),
       fetchTutorials: flow(fetchTutorials),
-      setActiveTutorial,
       resetActiveTutorial,
+      resetSeen,
+      setActiveTutorial,
       setMediaResources,
+      setSeenTime,
```

Comment:
I think you can remove this if you want to keep `setSeenTime` private.

Reply:
I'd think about this in the follow up PR. I got to wire this up to the UPP store next and I haven't decided exactly if I want the UPP to be updating the TutorialStore's seen state with what's in the UPP and if I want the TutorialStore observing UPP or have the UPP call this action on the TutorialStore when the UPP is loaded instead. If I have the UPP store call this directly, then it'll be public.

Reply:
No problem. I think this is good to merge then.